src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -61,16 +57,8 @@ func main() {
 		os.Exit(ConfigError)
 	}
 
-	if cfg.CpuProfile != "" {
-		startCPUProfiling(cfg.CpuProfile)
-		defer stopCPUProfiling(cfg.CpuProfile)
-	}
-
-	if cfg.MemProfile != "" {
-		defer saveMemProfile(cfg.MemProfile)
-	}
-
-	saveProfilingOnSigINT(cfg)
+	finishProfiling := profiling.Start(cfg.CpuProfile, cfg.MemProfile)
```

Comment:
I moved profiling into a new utility library that I can use in multiple programs.